benches/benches/block_target_gas_set/contract.rs

```diff
@@ -1,21 +1,9 @@
-use crate::{
-    utils::arb_dependent_cost_values,
-    *,
-};
+use crate::{utils::arb_dependent_cost_values, *};
```

Comment:
This likely needs a `cargo +nightly fmt`

Reply:
I think you're looking at an old version.